portal/src/main/webapp/js/src/oncoprint/webgl/src/js/oncoprintmodel.js

```diff
@@ -144,6 +145,10 @@ var OncoprintModel = (function () {
 	this.track_active_rules = {}; // from track id to active rule map (map with rule ids as keys)
 	this.track_info = {};
 	this.track_has_column_spacing = {}; // track id -> boolean
+	this.track_expandable = {}; // track id -> boolean
```

Comment:
`track_expansion_tracks` links a track to its already (visibly) existing tracks. This boolean, on the other hand, states whether a track option should be displayed to (run the expandCallback and) expand the track with more tracks.